src/main/java/frc/robot/lib/interfaces/Arm.java

```diff
@@ -20,68 +21,81 @@ public class Arm {
     private Timer timer = new Timer();
 
     public Arm() {
-        RobotMap.shoulderMotor.setNeutralMode(NeutralMode.Coast);
-        RobotMap.elbowMotor.setNeutralMode(NeutralMode.Coast);
-        RobotMap.wristMotor.setNeutralMode(NeutralMode.Coast);
-
-        RobotMap.wristMotor.setInverted(true);
-
-        RobotMap.shoulderMotor.configSelectedFeedbackSensor(TalonFXFeedbackDevice.IntegratedSensor, 0, 10);
-        RobotMap.elbowMotor.configSelectedFeedbackSensor(TalonFXFeedbackDevice.IntegratedSensor, 0, 10);
-        RobotMap.wristMotor.configSelectedFeedbackSensor(TalonFXFeedbackDevice.IntegratedSensor, 0, 10);
-
-        RobotMap.shoulderMotor.configNominalOutputForward(0, 10);
-        RobotMap.shoulderMotor.configNominalOutputReverse(0, 10);
-        RobotMap.shoulderMotor.configPeakOutputForward(1.0, 10);
-        RobotMap.shoulderMotor.configPeakOutputReverse(-1.0, 10);
-        RobotMap.shoulderMotor.configNeutralDeadband(0.001, 10);
-
-        RobotMap.elbowMotor.configNominalOutputForward(0, 10);
-        RobotMap.elbowMotor.configNominalOutputReverse(0, 10);
-        RobotMap.elbowMotor.configPeakOutputForward(1.0, 10);
-        RobotMap.elbowMotor.configPeakOutputReverse(-1.0, 10);
-        RobotMap.elbowMotor.configNeutralDeadband(0.001, 10);
-
-        RobotMap.wristMotor.configNominalOutputForward(0, 10);
-        RobotMap.wristMotor.configNominalOutputReverse(0, 10);
-        RobotMap.wristMotor.configPeakOutputForward(1.0, 10);
-        RobotMap.wristMotor.configPeakOutputReverse(-1.0, 10);
-        RobotMap.wristMotor.configNeutralDeadband(0.001, 10);
-
-        RobotMap.shoulderMotor.configAllowableClosedloopError(0, 0, 10);
-        RobotMap.elbowMotor.configAllowableClosedloopError(0, 0, 10);
-        RobotMap.wristMotor.configAllowableClosedloopError(0, 0, 10);
-
-        RobotMap.shoulderMotor.configForwardSoftLimitEnable(true);
-        RobotMap.elbowMotor.configForwardSoftLimitEnable(true);
-        RobotMap.wristMotor.configForwardSoftLimitEnable(true);
-
-        RobotMap.shoulderMotor.configForwardSoftLimitThreshold(Constants.SHOULDER_MOTOR.TRAVEL_LIMIT);
-        RobotMap.elbowMotor.configForwardSoftLimitThreshold(Constants.ELBOW_MOTOR.TRAVEL_LIMIT);
-        RobotMap.wristMotor.configForwardSoftLimitThreshold(Constants.WRIST_MOTOR.TRAVEL_LIMIT);
-
-        RobotMap.shoulderMotor.configReverseSoftLimitThreshold(-Constants.SHOULDER_MOTOR.TRAVEL_LIMIT);
-        RobotMap.elbowMotor.configReverseSoftLimitThreshold(-Constants.ELBOW_MOTOR.TRAVEL_LIMIT);
-        RobotMap.wristMotor.configReverseSoftLimitThreshold(-Constants.WRIST_MOTOR.TRAVEL_LIMIT);
+        RobotMap.leftShoulderMotor.setNeutralMode(NeutralMode.Coast);
```

Comment:
With the large number of motors involved this is starting to get a bit hard to read. I think we should start to get consistent with our groupings. Right now it is a mix of line groupings by function call and line groupings by variable name. Pick one and stick with it. So it should either be:

        RobotMap.leftShoulderMotor.setNeutralMode(NeutralMode.Coast);
        RobotMap.rightShoulderMotor.setNeutralMode(NeutralMode.Coast);
        RobotMap.leftElbowMotor.setNeutralMode(NeutralMode.Coast);
        RobotMap.rightElbowMotor.setNeutralMode(NeutralMode.Coast);
or:

        RobotMap.leftShoulderMotor.setNeutralMode(NeutralMode.Coast);
        RobotMap.leftShoulderMotor.configSelectedFeedbackSensor(TalonFXFeedbackDevice.IntegratedSensor, 0, 10);
        RobotMap.leftShoulderMotor.configNominalOutputForward(0, 10);
        RobotMap.leftShoulderMotor.configNominalOutputReverse(0, 10);
        RobotMap.leftShoulderMotor.configPeakOutputForward(1.0, 10);
        etc.